### PR TITLE
Add gitpod as an online to edit code

### PR DESCRIPTION
**Pull request by LinuxSuRen**

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
Wait for https://github.com/kubesphere/prow-tutorial/pull/41

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/kubesphere/s2i-java-container)
```
Comment (JohnNiang) [inline README.md]:
```diff
-kubesphere/s2i-java-container
+kubesphere/s2i-nodejs-container
```

**Review by JohnNiang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LinuxSuRen) [reply README.md]: My bad. Good review.

Comment (LinuxSuRen) [reply README.md]: This repo does not have the squash merge method setting. So, I just push the commit in the force way.

**Comment by LinuxSuRen** [comment]
We don't have an OWNERS file in the master branch. So I'm going to merge this manually.

**Pull request merged**